packages/flutter/lib/src/foundation/capabilities.dart

```diff
@@ -9,3 +9,13 @@ import '_capabilities_io.dart'
 ///
 /// Only to be used for web.
 bool get isCanvasKit => capabilities.isCanvasKit;
+
+/// Returns true if the application is using Skwasm.
+///
+/// Only to be used for web.
+bool get isSkwasm => capabilities.isSkwasm;
+
+/// Returns true if the application is using CanvasKit or Skwasm.
+///
+/// Only to be used for web.
+bool get isSkiaWeb => isCanvasKit || isSkwasm;
```

Comment:
This change is bloating the PR quite a bit (I suppose everywhere we check `isCanvasKit` now we check `isSkiaWeb`?)

I wonder if this would be much simpler if:

```
  isCanvasKit => capabilities.isCanvasKit || capabilities.isSkwasm;
```

And

```
  isSkwasm => capabilities.isSkwasm;
```

To separate the global "canvaskit" from the skwasm-specific cases?

I mean, the *capabilities* of the renderer are similar, aren't they?

Reply:
Essentially I'm doing a rename of the variable to be more accurate. I don't think we should have a variable called `isCanvasKit` that is true when the renderer is skwasm, that's just confusing. Hence, we have three separate explicit variables. Minimizing the number of lines changed in this particular PR is a non-goal, I'd rather the code be more clear.